user_manual/files/using_e2ee.rst

```diff
@@ -0,0 +1,277 @@
+
+   </details>
+
+
+.. raw:: html
+
+   <details>
+   <summary>iOS</summary>
+
+.. image:: ../images/e2ee-ios-setup.png
+    :width: 400px
+    :alt: Setup E2EE on iOS
+
+.. raw:: html
+
+   </details>
+
+
+.. raw:: html
+
+   <details>
+   <summary>Web</summary>
+
+.. warning::
+    It is not possible to enable E2EE in the browser.
+
+.. raw:: html
+
+   </details>
+
+
+.. raw:: html
+
+    <br />
+
+Encrypting a folder
+-----------------------
+
+.. warning::
+
+    You can only encrypt empty folders.
+
+
+.. raw:: html
+
+   <details>
+   <summary>Desktop</summary>
+
+.. image:: ../images/e2ee-desktop-encrypt.png
+    :width: 750px
+    :alt: Encrypting a folder on the desktop client
+
+.. raw:: html
+
+   </details>
+
+
+.. raw:: html
+
+   <details>
+   <summary>Android</summary>
+
+.. image:: ../images/e2ee-android-encrypt.png
+    :width: 400px
+    :alt: Encrypting a folder on an Android device
+
+.. raw:: html
+
+   </details>
+
+
+.. raw:: html
+
+   <details>
+   <summary>iOS</summary>
+
+.. image:: ../images/e2ee-ios-encrypt.png
+    :width: 400px
+    :alt: Encrypting a folder on an iOS device
+
+.. raw:: html
+
+   </details>
+
+
+.. raw:: html
+
+   <details>
+   <summary>Web</summary>
+
+.. warning::
+
+    It is not possible to encrypt a folder in the browser.
+
+.. raw:: html
+
+   </details>
+
+
+.. raw:: html
+
+    <br />
+
+Adding an E2EE device
+---------------------
+
+.. raw:: html
+
+   <details>
+   <summary>Desktop</summary>
+
+.. image:: ../images/e2ee-desktop-add.png
+    :width: 750px
+    :alt: Setup a new desktop client
+
+.. raw:: html
+
+   </details>
+
+
+.. raw:: html
+
+   <details>
+   <summary>Android</summary>
+
+.. image:: ../images/e2ee-android-add.png
+    :width: 400px
+    :alt: Setup a new Android device
+
+.. raw:: html
+
+   </details>
+
+
+.. raw:: html
+
+   <details>
+   <summary>iOS</summary>
+
+.. image:: ../images/e2ee-ios-add.png
+    :width: 400px
+    :alt: Setup a new iOS device
+
+.. raw:: html
+
+   </details>
+
+
+.. raw:: html
+
+   <details>
+   <summary>Web</summary>
+
+In the browser, you'll first have to enable E2EE in the personal settings. This needed, as E2EE is less secure in the browser as you need to fully trust your administrator to not alter the source code that your browser will execute.
+E2EE folders are currently read-only. Therefore, it is not possible to add, remove, edit, or share an E2EE file from the browser.
+
+.. image:: ../images/e2ee-web-add-1.png
+    :width: 750px
+    :alt: Setup a new browser session step 1
+
+.. image:: ../images/e2ee-web-add-2.png
+    :width: 750px
+    :alt: Setup a new browser session step 2
+
+.. image:: ../images/e2ee-web-add-3.png
+    :width: 750px
+    :alt: Setup a new browser session step 3
+
+.. raw:: html
+
+   </details>
+
+
+.. raw:: html
+
+    <br />
+
+
+Displaying the mnemonic
+-----------------------
+
+.. raw:: html
+
+   <details>
+   <summary>Desktop</summary>
+
+.. image:: ../images/e2ee-desktop-mnemonic.png
```

Comment:
- [ ] Black box used as censoring should simply be made the same colour as the UI.